src/index.html

```diff
@@ -1,15 +1,181 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form  action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <div class="block">
+          <label for="surname">
+            Surname:
+          </label>
+          <input type="text"
+                 name="surname"
+                 id="surname"
+                 autocomplete="off">
+        </div>
+        <div class="block">
+          <label for="name">
+              Name:
+          </label>
+          <input type="text"
+                   name="name"
+                   id="name">
+        </div>
+        <div class="block">
+          <label for="age">
+          How old are You?
+          </label>
+          <input type="number"
+                 name="age"
+                 id="age"
+                 value="12"
+                 min="12"
+                 max="100">
```

Comment:
```suggestion
          <input 
            type="number"
            name="age"
            id="age"
            value="12"
            min="12"
            max="100"
          >
```
исправь так везде где это нужно

Reply:
fixed